training/train.py

```diff
@@ -355,6 +373,7 @@ def train_model():
         save_obj(s3_obj, trained_recommender, precision_at_30, recall_at_30,
                  format_pkg_id_dict, id_package_dict, format_mnf_id_dict,
                  precision_at_50, recall_at_50, lower_limit, upper_limit, latent_factors)
+        _logger.info('Len of github token: {}'.format(len(GITHUB_TOKEN)))
```

Comment:
Is this log useful?

Reply:
I justified before, I will re-phrase them.

Reply:
I am really not sure about what value this is providing. maybe you can log if the GH token is not found.